scripts/buildSampleTree.ts

```diff
@@ -0,0 +1,75 @@
+import { toWei, toBN, toBNWei, getParamType, defaultAbiCoder, keccak256 } from "../test/utils";
```

Comment:
we could try generalize this more. I found this to be very useful when testing on kovan.